File: SproInstall/SocketPro/samples/RemoteSum/csharp/Client/RemoteSum_i.cs
public class RemoteSumConst
{
	//defines for service RemSum
	public const int sidRemSum = ((int)USOCKETLib.tagOtherDefine.odUserServiceIDMin + 10);

	public const short idDoSumRemSum = ((short)USOCKETLib.tagOtherDefine.odUserRequestIDMin + 0);
	public const short idPauseRemSum = (idDoSumRemSum + 1);
	public const short idRedoSumRemSum = (idPauseRemSum + 1);
	public const short idReportProgress = (idRedoSumRemSum + 1);
}

File: SproInstall/SocketPro/samples/RemoteSum/vbnet/Server/RemoteSum_i.vb
Public Class RemoteSumConst
	'defines for service RemSum
    Public Const sidRemSum As Integer = (USOCKETLib.tagOtherDefine.odUserServiceIDMin + 10)

    Public Const idDoSumRemSum As Short = (USOCKETLib.tagOtherDefine.odUserRequestIDMin + 0)
    Public Const idPauseRemSum As Short = (idDoSumRemSum + 1)
    Public Const idRedoSumRemSum As Short = (idPauseRemSum + 1)
	Public Const idReportProgress As Short = (idRedoSumRemSum + 1)
End Class